src/maintainer/knowledge_base.rst

```diff
@@ -1179,6 +1179,18 @@ For now, you will have to add ``nvcuda.dll`` to the ``missing_dso_whitelist``::
       - "*/nvcuda.dll"   # [win]
 
 
+My feedstock is not building old CUDA versions anymore
```

Comment:
Can you add a note that it's not recommended to this as this places a large burden on our CI resources and to do it only if there are known use cases for it.

Reply:
Done! Let me know what you think.